src/extlibs/hrvo/hrvo_agent.cpp

```diff
@@ -88,11 +87,13 @@ Agent::VelocityObstacle HRVOAgent::createVelocityObstacle(const Agent &other_age
             std::asin((radius_ + other_agent.getRadius()) /
                       (position_ - other_agent.getPosition()).length());
         // Direction of the two edges of the velocity obstacles
-        velocityObstacle.side1_ =
+        velocityObstacle.right_side =
             Vector::createFromAngle(Angle::fromRadians(angle - openingAngle));
-        velocityObstacle.side2_ =
+        velocityObstacle.left_side =
             Vector::createFromAngle(Angle::fromRadians(angle + openingAngle));
 
+        assert(velocityObstacle.right_side.isToTheRightOf(velocityObstacle.left_side));
```

Comment:
I think this was added for debugging. It can be removed now

Reply:
removed